jenkins/model_output_processing_Jenkinsfile

```diff
@@ -28,7 +28,7 @@ pipeline {
         // May need to actually use yesterday's date depending on what time of day this runs
         //sh 'DATE=$(date -d "yesterday 13:00" "+%Y_%m_%d"); wget https://owi-common-resources.s3-us-west-2.amazonaws.com/resources/application/nhm/output/data/climate_${DATE}.nc'
         sh '''
-          sh 'aws s3 sync s3://owi-common-resources/resources/application/nhm/output/data/ . --exclude "*" --include "*.nc"'
+          aws s3 sync s3://owi-common-resources/resources/application/nhm/output/data/ . --exclude "*" --include "*.nc"
           wget https://prod-owi-resources.s3-us-west-2.amazonaws.com/resources/Application/wbeep/model_output/all_quantiles.rds
           PATTERN="*.nc"
```

Comment:
If you move PATTERN to the top, you can use that variable in the aws s3 sync command for the include value. Jussayin...

Reply:
k

Reply:
do I need to quote it somehow or should I include that in my pattern if I need it (was: --include "*.nc" but now i presume PATTERN will come in without the doublequotes around it?

Reply:
See orevious comment around this :)

Reply:
Lookin good

Reply:
Sorry, got it.